### PR TITLE
fix(@desktop/wallet): Fix register ENS flow which is currently broken

### PR DESCRIPTION
**Pull request by Khushboo-dev-cpp**
### What does the PR do

Fixes the register Ens flow

### Affected areas

Register Ens, SendModal

### Architecture compliance

- [ ] I am familiar with the application architecture and agreed good practices.
My PR is consistent with this document: [Architecture guidelines](../architecture-guide.md)

### Screenshot of functionality (including design for comparison)

- [x] I've checked the design and this PR matches it

https://github.com/user-attachments/assets/33597402-ea8b-49b4-b442-1c50c7fcdab9

https://github.com/user-attachments/assets/db37671b-219c-48f8-a979-4d376f98780c

image

### CODE REVIEW

**Comment by caybro** [comment]
Is there an issue linked to this?

**Review by alaibe — APPROVED** [review]

File: ui/imports/shared/popups/send/SendModal.qml [hunk 1]
```diff
@@ -42,6 +42,10 @@ StatusDialog {
     property string preDefinedAmountToSend
     property string stickersPackId
 
+    // needed for registering ENS
+    property string profilePublicKey
```
Comment (saledjenic) [inline ui/imports/shared/popups/send/SendModal.qml:46]: profilePublicKey -> publicKey

File: ui/imports/shared/popups/send/SendModal.qml [hunk 2]
```diff
@@ -42,6 +42,10 @@ StatusDialog {
...
+    property string userName
```
Comment (saledjenic) [inline ui/imports/shared/popups/send/SendModal.qml:47]: userName -> ensName

**Review by saledjenic — COMMENTED** [review]
@Khushboo-dev-cpp could you split changes into two commits:
1. one with only `sTr` to `qsTr` changes, cause that somehow got missed for release branch
2. one with changes that fix ens flow

**Comment by Khushboo-dev-cpp** [comment]
> @Khushboo-dev-cpp could you split changes into two commits:
> 
> 1. one with only `sTr` to `qsTr` changes, cause that somehow got missed for release branch
> 2. one with changes that fix ens flow

not sure why its needed  though, it was missed in this cherry-pick  here https://github.com/status-im/status-desktop/pull/15865/files

**Comment by saledjenic** [comment]
> not sure why its needed though, it was missed in this cherry-pick here 

Simply, commits should be logically split due to clarity, easier code review, debugging, reverting changes...

**Comment by Khushboo-dev-cpp** [comment]
> > not sure why its needed though, it was missed in this cherry-pick here
> 
> Simply, commits should be logically split due to clarity, easier code review, debugging, reverting changes...
done!

**Review by saledjenic — APPROVED** [review]

**Review by caybro — APPROVED** [review]
LGTM

**Comment by saledjenic** [comment]
@Khushboo-dev-cpp could you cherry-pick the commit that is fixing broken ens flow to master?

**Comment by Khushboo-dev-cpp** [comment]
> @Khushboo-dev-cpp could you cherry-pick the commit that is fixing broken ens flow to master?

https://github.com/status-im/status-desktop/pull/16123 done here

**Pull request merged**